Dockerfile

```diff
@@ -37,12 +40,18 @@ RUN curl -k https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraf
     rm -f terraform_${TERRAFORM_VERSION}_linux_amd64.zip  && \
     rm -f terraform_${TERRAFORM_VERSION}_SHA256SUMS
 
-RUN pip install --no-cache-dir -r requirements.txt
+RUN pip install --no-cache-dir -r requirements.txt && \
```

Comment:
we should be able to have a 'before_script' stanza in the .gitlab-ci.yml instead of adding it here. If we must have it here, then we should add it to the requirements